### PR TITLE
weekly size update

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
PTAL @jridgewell

File: test/size.txt [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+  90.5 kB   |    15.63 kB   |    5.72 kB   |   v0/amp-accordion-0.1.js
+350.84 kB   |    94.83 kB   |   30.13 kB   |   v0/amp-analytics-0.1.js
+ 76.81 kB   |    11.17 kB   |    4.25 kB   |   v0/amp-anim-0.1.js
+ 73.63 kB   |    10.45 kB   |    4.12 kB   |   v0/amp-audio-0.1.js
+  68.4 kB   |     9.65 kB   |    3.61 kB   |   v0/amp-brid-player-0.1.js
+ 75.53 kB   |    11.42 kB   |     4.4 kB   |   v0/amp-brightcove-0.1.js
+206.93 kB   |    42.72 kB   |   12.57 kB   |   v0/amp-carousel-0.1.js
+  61.1 kB   |     7.56 kB   |    3.04 kB   |   v0/amp-dailymotion-0.1.js
+ 89.22 kB   |    11.77 kB   |    4.39 kB   |   v0/amp-dynamic-css-classes-0.1.js
+106.42 kB   |    18.13 kB   |    6.82 kB   |   v0/amp-facebook-0.1.js
```
Comment (jridgewell) [inline test/size.txt]: How'd this one change?

**Comment by jridgewell** [comment]
:shipit:

File: test/size.txt (hunk 1 above)
Comment (erwinmombay) [inline test/size.txt]: only thing i could think of is the `internalRuntimeToken` and `internalRuntimeVersion` being different on each `build/dist` pass.

**Pull request merged**